packages/shared/src/components/markdown.module.css

```diff
@@ -0,0 +1,125 @@
+.markdown {
+  & :where(h1) {
```

Comment:
Why do you use `& :where(x)` instead of `& x`?

Reply:
More stricter selector in terms of resetting the `:first-child` and `:last-child` and child components.
Also enables us to target multiple at once for instance the `ul, ol` one.

I've taking this approach from Tailwind, after finding issues with resetting  child types.

Reply:
Thanks for explaining :) Just make sure we have a proper browser support or that PostCSS compiles it to something every browser supports